<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Human-associated/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Human-associated - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">Reference</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Human-associated.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#human-associated" class="nav-link">Human-associated</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#combinations" class="nav-link">Combinations</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="human-associated">Human-associated</h1>
<p>human-associated</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Human-associated">mixs.vocab:Human-associated</a></p>
<!-- no inheritance hierarchy -->

<h2 id="combinations">Combinations</h2>
<ul>
<li><strong>Human-associated</strong><ul>
<li><a href="../Human-associatedMIGSEukaryote/">Human-associatedMIGSEukaryote</a> [ MIGS eukaryote]</li>
<li><a href="../Human-associatedMIGSBacteria/">Human-associatedMIGSBacteria</a> [ MIGS bacteria]</li>
<li><a href="../Human-associatedMIGSPlant/">Human-associatedMIGSPlant</a> [ MIGS plant]</li>
<li><a href="../Human-associatedMIGSVirus/">Human-associatedMIGSVirus</a> [ MIGS virus]</li>
<li><a href="../Human-associatedMIGSOrg/">Human-associatedMIGSOrg</a> [ MIGS org]</li>
<li><a href="../Human-associatedMIMS/">Human-associatedMIMS</a> [ MIMS]</li>
<li><a href="../Human-associatedMIMARKSSpecimen/">Human-associatedMIMARKSSpecimen</a> [ MIMARKS specimen]</li>
<li><a href="../Human-associatedMIMARKSSurvey/">Human-associatedMIMARKSSurvey</a> [ MIMARKS survey]</li>
<li><a href="../Human-associatedMISAG/">Human-associatedMISAG</a> [ MISAG]</li>
<li><a href="../Human-associatedMIMAG/">Human-associatedMIMAG</a> [ MIMAG]</li>
<li><a href="../Human-associatedMIUVIG/">Human-associatedMIUVIG</a> [ MIUVIG]</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>0..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>0..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>0..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>0..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>0..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>0..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000861/">0000861</a></td>
<td>host_subject_id</td>
<td>0..1</td>
<td>A unique identifier by which each subject can be referred to, de-identified.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000255/">0000255</a></td>
<td>host_age</td>
<td>0..1</td>
<td>Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000811/">0000811</a></td>
<td>host_sex</td>
<td>0..1</td>
<td>Gender or physical sex of the host.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000884/">0000884</a></td>
<td>ihmc_medication_code</td>
<td>0..*</td>
<td>Can include multiple medication codes</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000751/">0000751</a></td>
<td>chem_administration</td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
</tr>
<tr>
<td><a href="../0000867/">0000867</a></td>
<td>host_body_site</td>
<td>0..1</td>
<td>Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000888/">0000888</a></td>
<td>host_body_product</td>
<td>0..1</td>
<td>Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000263/">0000263</a></td>
<td>host_tot_mass</td>
<td>0..1</td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000264/">0000264</a></td>
<td>host_height</td>
<td>0..1</td>
<td>The height of subject</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000869/">0000869</a></td>
<td>host_diet</td>
<td>0..*</td>
<td>Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000870/">0000870</a></td>
<td>host_last_meal</td>
<td>0..*</td>
<td>Content of last meal and time since feeding; can include multiple values</td>
<td>{text};{duration}</td>
</tr>
<tr>
<td><a href="../0000872/">0000872</a></td>
<td>host_family_relation</td>
<td>0..*</td>
<td>Familial relationships to other hosts in the same study; can include multiple relationships</td>
<td>{text};{text}</td>
</tr>
<tr>
<td><a href="../0000365/">0000365</a></td>
<td>host_genotype</td>
<td>0..1</td>
<td>Observed genotype</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000874/">0000874</a></td>
<td>host_phenotype</td>
<td>0..1</td>
<td>Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000274/">0000274</a></td>
<td>host_body_temp</td>
<td>0..1</td>
<td>Core body temperature of the host when sample was collected</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000262/">0000262</a></td>
<td>smoker</td>
<td>0..1</td>
<td>Specification of smoking status</td>
<td>{boolean}</td>
</tr>
<tr>
<td><a href="../0000265/">0000265</a></td>
<td>host_hiv_stat</td>
<td>0..1</td>
<td>HIV status of subject, if yes HAART initiation status should also be indicated as [YES or NO]</td>
<td>{boolean};{boolean}</td>
</tr>
<tr>
<td><a href="../0000894/">0000894</a></td>
<td>drug_usage</td>
<td>0..*</td>
<td>Any drug used by subject and the frequency of usage; can include multiple drugs used</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000317/">0000317</a></td>
<td>host_body_mass_index</td>
<td>0..1</td>
<td>Body mass index, calculated as weight/(height)squared</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000266/">0000266</a></td>
<td>diet_last_six_month</td>
<td>0..1</td>
<td>Specification of major diet changes in the last six months, if yes the change should be specified</td>
<td>{boolean};{text}</td>
</tr>
<tr>
<td><a href="../0000295/">0000295</a></td>
<td>weight_loss_3_month</td>
<td>0..1</td>
<td>Specification of weight loss in the last three months, if yes should be further specified to include amount of weight loss</td>
<td>{boolean};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000895/">0000895</a></td>
<td>ethnicity</td>
<td>0..*</td>
<td>A category of people who identify with each other, usually on the basis of presumed similarities such as a common language, ancestry, history, society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000896/">0000896</a></td>
<td>host_occupation</td>
<td>0..1</td>
<td>Most frequent job performed by subject</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000267/">0000267</a></td>
<td>pet_farm_animal</td>
<td>0..*</td>
<td>Specification of presence of pets or farm animals in the environment of subject, if yes the animals should be specified; can include multiple animals present</td>
<td>{boolean};{text}</td>
</tr>
<tr>
<td><a href="../0000268/">0000268</a></td>
<td>travel_out_six_month</td>
<td>0..*</td>
<td>Specification of the countries travelled in the last six months; can include multiple travels</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000326/">0000326</a></td>
<td>twin_sibling</td>
<td>0..1</td>
<td>Specification of twin sibling presence</td>
<td>{boolean}</td>
</tr>
<tr>
<td><a href="../0000897/">0000897</a></td>
<td>medic_hist_perform</td>
<td>0..1</td>
<td>Whether full medical history was collected</td>
<td>{boolean}</td>
</tr>
<tr>
<td><a href="../0000898/">0000898</a></td>
<td>study_complt_stat</td>
<td>0..1</td>
<td>Specification of study completion status, if no the reason should be specified</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000269/">0000269</a></td>
<td>pulmonary_disord</td>
<td>0..*</td>
<td>History of pulmonary disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000270/">0000270</a></td>
<td>nose_throat_disord</td>
<td>0..*</td>
<td>History of nose-throat disorders; can include multiple disorders,  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory tract disease (https://disease-ontology.org/?id=DOID:974).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000271/">0000271</a></td>
<td>blood_blood_disord</td>
<td>0..*</td>
<td>History of blood disorders; can include multiple disorders.  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, hematopoietic system disease (https://disease-ontology.org/?id=DOID:74).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000333/">0000333</a></td>
<td>host_pulse</td>
<td>0..1</td>
<td>Resting pulse, measured as beats per minute</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000272/">0000272</a></td>
<td>gestation_state</td>
<td>0..1</td>
<td>Specification of the gestation state</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000273/">0000273</a></td>
<td>maternal_health_stat</td>
<td>0..1</td>
<td>Specification of the maternal health status</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000275/">0000275</a></td>
<td>foetal_health_stat</td>
<td>0..1</td>
<td>Specification of foetal health status, should also include abortion</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000276/">0000276</a></td>
<td>amniotic_fluid_color</td>
<td>0..1</td>
<td>Specification of the color of the amniotic fluid sample</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000277/">0000277</a></td>
<td>kidney_disord</td>
<td>0..*</td>
<td>History of kidney disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, kidney disease (https://disease-ontology.org/?id=DOID:557).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000278/">0000278</a></td>
<td>urogenit_tract_disor</td>
<td>0..*</td>
<td>History of urogenital tract disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, urinary system disease (https://disease-ontology.org/?id=DOID:18).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000899/">0000899</a></td>
<td>urine_collect_meth</td>
<td>0..1</td>
<td>Specification of urine collection method</td>
<td>[clean catch</td>
</tr>
<tr>
<td><a href="../0000754/">0000754</a></td>
<td>perturbation</td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000183/">0000183</a></td>
<td>salinity</td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000753/">0000753</a></td>
<td>oxy_stat_samp</td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000103/">0000103</a></td>
<td>organism_count</td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000110/">0000110</a></td>
<td>samp_store_temp</td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000116/">0000116</a></td>
<td>samp_store_dur</td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0001298/">0001298</a></td>
<td>host_symbiont</td>
<td>0..*</td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000755/">0000755</a></td>
<td>samp_store_loc</td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
