<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/AirMIMARKSSurvey/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>AirMIMARKSSurvey - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">Reference</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/AirMIMARKSSurvey.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#airmimarkssurvey" class="nav-link">AirMIMARKSSurvey</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="airmimarkssurvey">AirMIMARKSSurvey</h1>
<p>Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package air</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/AirMIMARKSSurvey">mixs.vocab:AirMIMARKSSurvey</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Air/">Air</a><ul>
<li><strong>AirMIMARKSSurvey</strong> [ MIMARKS survey]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">0001107</a></td>
<td>samp_name</td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001320/">0001320</a></td>
<td>samp_taxon_id</td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
</tr>
<tr>
<td><a href="../0000092/">0000092</a></td>
<td>project_name</td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000008/">0000008</a></td>
<td>experimental_factor</td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001321/">0001321</a></td>
<td>neg_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001322/">0001322</a></td>
<td>pos_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000020/">0000020</a></td>
<td>subspecf_gen_lin</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
</tr>
<tr>
<td><a href="../0000023/">0000023</a></td>
<td>extrachrom_elements</td>
<td>0..1</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000026/">0000026</a></td>
<td>source_mat_id</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000028/">0000028</a></td>
<td>biotic_relationship</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000032/">0000032</a></td>
<td>trophic_level</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000015/">0000015</a></td>
<td>rel_to_oxygen</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000003/">0000003</a></td>
<td>isol_growth_condt</td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collec_device</td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collec_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000016/">0000016</a></td>
<td>samp_mat_process</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000001/">0000001</a></td>
<td>samp_size</td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000111/">0000111</a></td>
<td>samp_vol_we_dna_ext</td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000037/">0000037</a></td>
<td>nucl_acid_ext</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000038/">0000038</a></td>
<td>nucl_acid_amp</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000044/">0000044</a></td>
<td>target_gene</td>
<td>1..1</td>
<td>Targeted gene or locus name for marker gene studies</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000045/">0000045</a></td>
<td>target_subfragment</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000046/">0000046</a></td>
<td>pcr_primers</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</td>
<td>FWD:{dna};REV:{dna}</td>
</tr>
<tr>
<td><a href="../0000049/">0000049</a></td>
<td>pcr_cond</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'</td>
<td>initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final elongation:degrees_minutes;total cycles</td>
</tr>
<tr>
<td><a href="../0000050/">0000050</a></td>
<td>seq_meth</td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000051/">0000051</a></td>
<td>seq_quality_check</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</td>
<td>[none</td>
</tr>
<tr>
<td><a href="../0000052/">0000052</a></td>
<td>chimera_check</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool(s) used for chimera checking, including version number and parameters, to discover and remove chimeric sequences. A chimeric sequence is comprised of two or more phylogenetically distinct parent sequences.</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>associated resource</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000090/">0000090</a></td>
<td>sop</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>1..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000096/">0000096</a></td>
<td>barometric_press</td>
<td>0..1</td>
<td>Force per unit area exerted against a surface by the weight of air above that surface</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000097/">0000097</a></td>
<td>carb_dioxide</td>
<td>0..1</td>
<td>Carbon dioxide (gas) amount or concentration at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000098/">0000098</a></td>
<td>carb_monoxide</td>
<td>0..1</td>
<td>Carbon monoxide (gas) amount or concentration at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000751/">0000751</a></td>
<td>chem_administration</td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
</tr>
<tr>
<td><a href="../0000100/">0000100</a></td>
<td>humidity</td>
<td>0..1</td>
<td>Amount of water vapour in the air, at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000101/">0000101</a></td>
<td>methane</td>
<td>0..1</td>
<td>Methane (gas) amount or concentration at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000103/">0000103</a></td>
<td>organism_count</td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000104/">0000104</a></td>
<td>oxygen</td>
<td>0..1</td>
<td>Oxygen (gas) amount or concentration at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000753/">0000753</a></td>
<td>oxy_stat_samp</td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000754/">0000754</a></td>
<td>perturbation</td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000107/">0000107</a></td>
<td>pollutants</td>
<td>0..*</td>
<td>Pollutant types and, amount or concentrations measured at the time of sampling; can report multiple pollutants by entering numeric values preceded by name of pollutant</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000108/">0000108</a></td>
<td>air particulate matter concentration</td>
<td>0..*</td>
<td>Concentration of substances that remain suspended in the air, and comprise mixtures of organic and inorganic substances (PM10 and PM2.5); can report multiple PM's by entering numeric values preceded by name of PM</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000183/">0000183</a></td>
<td>salinity</td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000116/">0000116</a></td>
<td>samp_store_dur</td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0000755/">0000755</a></td>
<td>samp_store_loc</td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000110/">0000110</a></td>
<td>samp_store_temp</td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000112/">0000112</a></td>
<td>solar_irradiance</td>
<td>0..1</td>
<td>The amount of solar energy that arrives at a specific area of a surface during a specific time interval</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000114/">0000114</a></td>
<td>ventilation_rate</td>
<td>0..1</td>
<td>Ventilation rate of the system in the sampled premises</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000756/">0000756</a></td>
<td>ventilation_type</td>
<td>0..1</td>
<td>Ventilation system used in the sampled premises</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000115/">0000115</a></td>
<td>volatile_org_comp</td>
<td>0..*</td>
<td>Concentration of carbon-based chemicals that easily evaporate at room temperature; can report multiple volatile organic compounds by entering numeric values preceded by name of compound</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000757/">0000757</a></td>
<td>wind_direction</td>
<td>0..1</td>
<td>Wind direction is the direction from which a wind originates</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000118/">0000118</a></td>
<td>wind_speed</td>
<td>0..1</td>
<td>Speed of wind measured at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
