<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/SoilMIGSBacteria/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>SoilMIGSBacteria - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">Reference</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/SoilMIGSBacteria.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#soilmigsbacteria" class="nav-link">SoilMIGSBacteria</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="soilmigsbacteria">SoilMIGSBacteria</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package soil</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/SoilMIGSBacteria">mixs.vocab:SoilMIGSBacteria</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Soil/">Soil</a><ul>
<li><strong>SoilMIGSBacteria</strong> [ MIGS bacteria]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">0001107</a></td>
<td>samp_name</td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001320/">0001320</a></td>
<td>samp_taxon_id</td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
</tr>
<tr>
<td><a href="../0000092/">0000092</a></td>
<td>project_name</td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000008/">0000008</a></td>
<td>experimental_factor</td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001321/">0001321</a></td>
<td>neg_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001322/">0001322</a></td>
<td>pos_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000020/">0000020</a></td>
<td>subspecf_gen_lin</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
</tr>
<tr>
<td><a href="../0000022/">0000022</a></td>
<td>num_replicons</td>
<td>1..1</td>
<td>Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000023/">0000023</a></td>
<td>extrachrom_elements</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000024/">0000024</a></td>
<td>estimated_size</td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
</tr>
<tr>
<td><a href="../0000025/">0000025</a></td>
<td>ref_biomaterial</td>
<td>1..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000026/">0000026</a></td>
<td>source_mat_id</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000027/">0000027</a></td>
<td>pathogenicity</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>To what is the entity pathogenic</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000028/">0000028</a></td>
<td>biotic_relationship</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000029/">0000029</a></td>
<td>specific_host</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000030/">0000030</a></td>
<td>host_spec_range</td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000031/">0000031</a></td>
<td>host_disease_stat</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000032/">0000032</a></td>
<td>trophic_level</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000034/">0000034</a></td>
<td>encoded_traits</td>
<td>0..1</td>
<td>Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000015/">0000015</a></td>
<td>rel_to_oxygen</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000003/">0000003</a></td>
<td>isol_growth_condt</td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collec_device</td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collec_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000016/">0000016</a></td>
<td>samp_mat_process</td>
<td>0..1</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000001/">0000001</a></td>
<td>samp_size</td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000111/">0000111</a></td>
<td>samp_vol_we_dna_ext</td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000037/">0000037</a></td>
<td>nucl_acid_ext</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000038/">0000038</a></td>
<td>nucl_acid_amp</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000039/">0000039</a></td>
<td>lib_size</td>
<td>0..1</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000040/">0000040</a></td>
<td>lib_reads_seqd</td>
<td>0..1</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000041/">0000041</a></td>
<td>lib_layout</td>
<td>0..1</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000042/">0000042</a></td>
<td>lib_vector</td>
<td>0..1</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000043/">0000043</a></td>
<td>lib_screen</td>
<td>0..1</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000048/">0000048</a></td>
<td>adapters</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
</tr>
<tr>
<td><a href="../0000050/">0000050</a></td>
<td>seq_meth</td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000053/">0000053</a></td>
<td>tax_ident</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000056/">0000056</a></td>
<td>assembly_qual</td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000057/">0000057</a></td>
<td>assembly_name</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
</tr>
<tr>
<td><a href="../0000058/">0000058</a></td>
<td>assembly_software</td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000059/">0000059</a></td>
<td>annot</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000060/">0000060</a></td>
<td>number_contig</td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000061/">0000061</a></td>
<td>feat_pred</td>
<td>0..1</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000062/">0000062</a></td>
<td>ref_db</td>
<td>0..1</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
</tr>
<tr>
<td><a href="../0000063/">0000063</a></td>
<td>sim_search_meth</td>
<td>0..1</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000064/">0000064</a></td>
<td>tax_class</td>
<td>0..1</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000069/">0000069</a></td>
<td>compl_score</td>
<td>0..1</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000070/">0000070</a></td>
<td>compl_software</td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>associated resource</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000090/">0000090</a></td>
<td>sop</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>1..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>1..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001080/">0001080</a></td>
<td>cur_land_use</td>
<td>0..1</td>
<td>Present state of sample site</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000312/">0000312</a></td>
<td>cur_vegetation</td>
<td>0..1</td>
<td>Vegetation classification from one or more standard classification systems, or agricultural crop</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000314/">0000314</a></td>
<td>cur_vegetation_meth</td>
<td>0..1</td>
<td>Reference or method used in vegetation classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000315/">0000315</a></td>
<td>previous_land_use</td>
<td>0..1</td>
<td>Previous land use and dates</td>
<td>{text};{timestamp}</td>
</tr>
<tr>
<td><a href="../0000316/">0000316</a></td>
<td>prev_land_use_meth</td>
<td>0..1</td>
<td>Reference or method used in determining previous land use and dates</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000318/">0000318</a></td>
<td>crop_rotation</td>
<td>0..1</td>
<td>Whether or not crop is rotated, and if yes, rotation schedule</td>
<td>{boolean};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000639/">0000639</a></td>
<td>agrochem_addition</td>
<td>0..*</td>
<td>Addition of fertilizers, pesticides, etc. - amount and time of applications</td>
<td>{text};{float} {unit};{timestamp}</td>
</tr>
<tr>
<td><a href="../0001081/">0001081</a></td>
<td>tillage</td>
<td>0..*</td>
<td>Note method(s) used for tilling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001086/">0001086</a></td>
<td>fire</td>
<td>0..1</td>
<td>Historical and/or physical evidence of fire</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000319/">0000319</a></td>
<td>flooding</td>
<td>0..1</td>
<td>Historical and/or physical evidence of flooding</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000320/">0000320</a></td>
<td>extreme_event</td>
<td>0..1</td>
<td>Unusual physical events that may have affected microbial populations</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001082/">0001082</a></td>
<td>soil_horizon</td>
<td>0..1</td>
<td>Specific layer in the land area which measures parallel to the soil surface and possesses physical characteristics which differ from the layers above and beneath</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000321/">0000321</a></td>
<td>horizon_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the horizon</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000322/">0000322</a></td>
<td>sieving</td>
<td>0..1</td>
<td>Collection design of pooled samples and/or sieve size and amount of sample sieved</td>
<td>{{text}</td>
</tr>
<tr>
<td><a href="../0000185/">0000185</a></td>
<td>water_content</td>
<td>0..1</td>
<td>Water content measurement</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000323/">0000323</a></td>
<td>water_cont_soil_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the water content of soil</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000325/">0000325</a></td>
<td>pool_dna_extracts</td>
<td>0..1</td>
<td>Indicate whether multiple DNA extractions were mixed. If the answer yes, the number of extracts that were pooled should be given</td>
<td>{boolean};{integer}</td>
</tr>
<tr>
<td><a href="../0000327/">0000327</a></td>
<td>store_cond</td>
<td>0..1</td>
<td>Explain how and for how long the soil sample was stored before DNA extraction (fresh/frozen/other).</td>
<td>{text};{duration}</td>
</tr>
<tr>
<td><a href="../0000328/">0000328</a></td>
<td>link_climate_info</td>
<td>0..1</td>
<td>Link to climate resource</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000642/">0000642</a></td>
<td>annual_temp</td>
<td>0..1</td>
<td>Mean annual temperature</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000643/">0000643</a></td>
<td>season_temp</td>
<td>0..1</td>
<td>Mean seasonal temperature</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000644/">0000644</a></td>
<td>annual_precpt</td>
<td>0..1</td>
<td>The average of all annual precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000645/">0000645</a></td>
<td>season_precpt</td>
<td>0..1</td>
<td>The average of all seasonal precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000329/">0000329</a></td>
<td>link_class_info</td>
<td>0..1</td>
<td>Link to digitized soil maps or other soil classification information</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001083/">0001083</a></td>
<td>fao_class</td>
<td>0..1</td>
<td>Soil classification from the FAO World Reference Database for Soil Resources. The list can be found at http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000330/">0000330</a></td>
<td>local_class</td>
<td>0..1</td>
<td>Soil classification based on local soil classification system</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000331/">0000331</a></td>
<td>local_class_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the local soil classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000205/">0000205</a></td>
<td>org_nitro</td>
<td>0..1</td>
<td>Concentration of organic nitrogen</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000332/">0000332</a></td>
<td>soil_type</td>
<td>0..1</td>
<td>Description of the soil type or classification. This field accepts terms under soil (http://purl.obolibrary.org/obo/ENVO_00001998).  Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000334/">0000334</a></td>
<td>soil_type_meth</td>
<td>0..1</td>
<td>Reference or method used in determining soil series name or other lower-level classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000646/">0000646</a></td>
<td>slope_gradient</td>
<td>0..1</td>
<td>Commonly called 'slope'. The angle between ground surface and a horizontal line (in percent). This is the direction that overland water would flow. This measure is usually taken with a hand level meter or clinometer</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000647/">0000647</a></td>
<td>slope_aspect</td>
<td>0..1</td>
<td>The direction a slope faces. While looking down a slope use a compass to record the direction you are facing (direction or degrees); e.g., nw or 315 degrees. This measure provides an indication of sun and wind exposure that will influence soil temperature and evapotranspiration.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001084/">0001084</a></td>
<td>profile_position</td>
<td>0..1</td>
<td>Cross-sectional position in the hillslope where sample was collected.sample area position in relation to surrounding areas</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001085/">0001085</a></td>
<td>drainage_class</td>
<td>0..1</td>
<td>Drainage classification from a standard system such as the USDA system</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000335/">0000335</a></td>
<td>soil_text_measure</td>
<td>0..1</td>
<td>The relative proportion of different grain sizes of mineral particles in a soil, as described using a standard system; express as % sand (50 um to 2 mm), silt (2 um to 50 um), and clay (&lt;2 um) with textural name (e.g., silty clay loam) optional.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000336/">0000336</a></td>
<td>soil_texture_meth</td>
<td>0..1</td>
<td>Reference or method used in determining soil texture</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001001/">0001001</a></td>
<td>ph</td>
<td>0..1</td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001106/">0001106</a></td>
<td>ph_meth</td>
<td>0..1</td>
<td>Reference or method used in determining ph</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000204/">0000204</a></td>
<td>org_matter</td>
<td>0..1</td>
<td>Concentration of organic matter</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000533/">0000533</a></td>
<td>tot_org_carb</td>
<td>0..1</td>
<td>Definition for soil: total organic carbon content of the soil, definition otherwise: total organic carbon content</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000337/">0000337</a></td>
<td>tot_org_c_meth</td>
<td>0..1</td>
<td>Reference or method used in determining total organic carbon</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000530/">0000530</a></td>
<td>tot_nitro_content</td>
<td>0..1</td>
<td>Total nitrogen content of the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000338/">0000338</a></td>
<td>tot_nitro_cont_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the total nitrogen</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000650/">0000650</a></td>
<td>microbial_biomass</td>
<td>0..1</td>
<td>The part of the organic matter in the soil that constitutes living microorganisms smaller than 5-10 micrometer. If you keep this, you would need to have correction factors used for conversion to the final units</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000339/">0000339</a></td>
<td>micro_biomass_meth</td>
<td>0..1</td>
<td>Reference or method used in determining microbial biomass</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000340/">0000340</a></td>
<td>link_addit_analys</td>
<td>0..1</td>
<td>Link to additional analysis results performed on the sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000341/">0000341</a></td>
<td>salinity_meth</td>
<td>0..1</td>
<td>Reference or method used in determining salinity</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000652/">0000652</a></td>
<td>heavy_metals</td>
<td>0..*</td>
<td>Heavy metals present in the sequenced sample and their concentrations. For multiple heavy metals and concentrations, add multiple copies of this field.</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000343/">0000343</a></td>
<td>heavy_metals_meth</td>
<td>0..1</td>
<td>Reference or method used in determining heavy metals</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000607/">0000607</a></td>
<td>al_sat</td>
<td>0..1</td>
<td>Aluminum saturation (esp. For tropical soils)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000324/">0000324</a></td>
<td>al_sat_meth</td>
<td>0..1</td>
<td>Reference or method used in determining Al saturation</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
