
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.3.7">
    
    
      
        <title>MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.4a0965b7.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#miscellaneousnaturalorartificialenvironmentmimarkssurvey" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Reference
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey.md" title="Edit this page" class="md-content__button md-icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>



<h1 id="miscellaneousnaturalorartificialenvironmentmimarkssurvey">MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey</h1>
<p>Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package miscellaneous natural or artificial environment</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey">mixs.vocab:MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../MiscellaneousNaturalOrArtificialEnvironment/">MiscellaneousNaturalOrArtificialEnvironment</a><ul>
<li><strong>MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey</strong> [ MIMARKS survey]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">0001107</a></td>
<td>samp_name</td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001320/">0001320</a></td>
<td>samp_taxon_id</td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
</tr>
<tr>
<td><a href="../0000092/">0000092</a></td>
<td>project_name</td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000008/">0000008</a></td>
<td>experimental_factor</td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001321/">0001321</a></td>
<td>neg_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001322/">0001322</a></td>
<td>pos_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000020/">0000020</a></td>
<td>subspecf_gen_lin</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
</tr>
<tr>
<td><a href="../0000023/">0000023</a></td>
<td>extrachrom_elements</td>
<td>0..1</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000026/">0000026</a></td>
<td>source_mat_id</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000028/">0000028</a></td>
<td>biotic_relationship</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000032/">0000032</a></td>
<td>trophic_level</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000015/">0000015</a></td>
<td>rel_to_oxygen</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000003/">0000003</a></td>
<td>isol_growth_condt</td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collec_device</td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collec_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000016/">0000016</a></td>
<td>samp_mat_process</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000001/">0000001</a></td>
<td>samp_size</td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000111/">0000111</a></td>
<td>samp_vol_we_dna_ext</td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000037/">0000037</a></td>
<td>nucl_acid_ext</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000038/">0000038</a></td>
<td>nucl_acid_amp</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000044/">0000044</a></td>
<td>target_gene</td>
<td>1..1</td>
<td>Targeted gene or locus name for marker gene studies</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000045/">0000045</a></td>
<td>target_subfragment</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000046/">0000046</a></td>
<td>pcr_primers</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</td>
<td>FWD:{dna};REV:{dna}</td>
</tr>
<tr>
<td><a href="../0000049/">0000049</a></td>
<td>pcr_cond</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'</td>
<td>initial denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final elongation:degrees_minutes;total cycles</td>
</tr>
<tr>
<td><a href="../0000050/">0000050</a></td>
<td>seq_meth</td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000051/">0000051</a></td>
<td>seq_quality_check</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</td>
<td>[none</td>
</tr>
<tr>
<td><a href="../0000052/">0000052</a></td>
<td>chimera_check</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool(s) used for chimera checking, including version number and parameters, to discover and remove chimeric sequences. A chimeric sequence is comprised of two or more phylogenetically distinct parent sequences.</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>associated resource</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000090/">0000090</a></td>
<td>sop</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000421/">0000421</a></td>
<td>alkalinity</td>
<td>0..1</td>
<td>Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000427/">0000427</a></td>
<td>ammonium</td>
<td>0..1</td>
<td>Concentration of ammonium in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000174/">0000174</a></td>
<td>biomass</td>
<td>0..*</td>
<td>Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000176/">0000176</a></td>
<td>bromide</td>
<td>0..1</td>
<td>Concentration of bromide</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000432/">0000432</a></td>
<td>calcium</td>
<td>0..1</td>
<td>Concentration of calcium in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000751/">0000751</a></td>
<td>chem_administration</td>
<td>0..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
</tr>
<tr>
<td><a href="../0000429/">0000429</a></td>
<td>chloride</td>
<td>0..1</td>
<td>Concentration of chloride in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000177/">0000177</a></td>
<td>chlorophyll</td>
<td>0..1</td>
<td>Concentration of chlorophyll</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000435/">0000435</a></td>
<td>density</td>
<td>0..1</td>
<td>Density of the sample, which is its mass per unit volume (aka volumetric mass density)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000178/">0000178</a></td>
<td>diether_lipids</td>
<td>0..*</td>
<td>Concentration of diether lipids; can include multiple types of diether lipids</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000436/">0000436</a></td>
<td>diss_carb_dioxide</td>
<td>0..1</td>
<td>Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000179/">0000179</a></td>
<td>diss_hydrogen</td>
<td>0..1</td>
<td>Concentration of dissolved hydrogen</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000434/">0000434</a></td>
<td>diss_inorg_carb</td>
<td>0..1</td>
<td>Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000162/">0000162</a></td>
<td>diss_org_nitro</td>
<td>0..1</td>
<td>Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000119/">0000119</a></td>
<td>diss_oxygen</td>
<td>0..1</td>
<td>Concentration of dissolved oxygen</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000425/">0000425</a></td>
<td>nitrate</td>
<td>0..1</td>
<td>Concentration of nitrate in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000426/">0000426</a></td>
<td>nitrite</td>
<td>0..1</td>
<td>Concentration of nitrite in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000504/">0000504</a></td>
<td>nitro</td>
<td>0..1</td>
<td>Concentration of nitrogen (total)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000508/">0000508</a></td>
<td>org_carb</td>
<td>0..1</td>
<td>Concentration of organic carbon</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000204/">0000204</a></td>
<td>org_matter</td>
<td>0..1</td>
<td>Concentration of organic matter</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000205/">0000205</a></td>
<td>org_nitro</td>
<td>0..1</td>
<td>Concentration of organic nitrogen</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000103/">0000103</a></td>
<td>organism_count</td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000753/">0000753</a></td>
<td>oxy_stat_samp</td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001001/">0001001</a></td>
<td>ph</td>
<td>0..1</td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000754/">0000754</a></td>
<td>perturbation</td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000505/">0000505</a></td>
<td>phosphate</td>
<td>0..1</td>
<td>Concentration of phosphate</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000181/">0000181</a></td>
<td>phosplipid_fatt_acid</td>
<td>0..*</td>
<td>Concentration of phospholipid fatty acids; can include multiple values</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000430/">0000430</a></td>
<td>potassium</td>
<td>0..1</td>
<td>Concentration of potassium in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000412/">0000412</a></td>
<td>pressure</td>
<td>0..1</td>
<td>Pressure to which the sample is subject to, in atmospheres</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000183/">0000183</a></td>
<td>salinity</td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000116/">0000116</a></td>
<td>samp_store_dur</td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0000755/">0000755</a></td>
<td>samp_store_loc</td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000110/">0000110</a></td>
<td>samp_store_temp</td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000184/">0000184</a></td>
<td>silicate</td>
<td>0..1</td>
<td>Concentration of silicate</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000428/">0000428</a></td>
<td>sodium</td>
<td>0..1</td>
<td>Sodium concentration in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000423/">0000423</a></td>
<td>sulfate</td>
<td>0..1</td>
<td>Concentration of sulfate in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000424/">0000424</a></td>
<td>sulfide</td>
<td>0..1</td>
<td>Concentration of sulfide in the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000203/">0000203</a></td>
<td>water_current</td>
<td>0..1</td>
<td>Measurement of magnitude and direction of flow within a fluid</td>
<td>None</td>
</tr>
</tbody>
</table>

              
            </article>
            
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.b97dbffb.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.0238f547.min.js"></script>
      
    
  </body>
</html>