<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Food-humanFoodsMIGSEukaryote/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Food-humanFoodsMIGSEukaryote - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">Reference</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Food-humanFoodsMIGSEukaryote.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#food-humanfoodsmigseukaryote" class="nav-link">Food-humanFoodsMIGSEukaryote</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="food-humanfoodsmigseukaryote">Food-humanFoodsMIGSEukaryote</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package food-human foods</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Food-humanFoodsMIGSEukaryote">mixs.vocab:Food-humanFoodsMIGSEukaryote</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Food-humanFoods/">Food-humanFoods</a><ul>
<li><strong>Food-humanFoodsMIGSEukaryote</strong> [ MIGS eukaryote]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">0001107</a></td>
<td>samp_name</td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001320/">0001320</a></td>
<td>samp_taxon_id</td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
</tr>
<tr>
<td><a href="../0000092/">0000092</a></td>
<td>project_name</td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000008/">0000008</a></td>
<td>experimental_factor</td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001321/">0001321</a></td>
<td>neg_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001322/">0001322</a></td>
<td>pos_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000020/">0000020</a></td>
<td>subspecf_gen_lin</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
</tr>
<tr>
<td><a href="../0000021/">0000021</a></td>
<td>ploidy</td>
<td>0..1</td>
<td>The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer to http://purl.bioontology.org/ontology/PATO</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000022/">0000022</a></td>
<td>num_replicons</td>
<td>0..1</td>
<td>Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000023/">0000023</a></td>
<td>extrachrom_elements</td>
<td>0..1</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000024/">0000024</a></td>
<td>estimated_size</td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
</tr>
<tr>
<td><a href="../0000025/">0000025</a></td>
<td>ref_biomaterial</td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000026/">0000026</a></td>
<td>source_mat_id</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000027/">0000027</a></td>
<td>pathogenicity</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>To what is the entity pathogenic</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000028/">0000028</a></td>
<td>biotic_relationship</td>
<td>0..1</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000029/">0000029</a></td>
<td>specific_host</td>
<td>0..1</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000030/">0000030</a></td>
<td>host_spec_range</td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000031/">0000031</a></td>
<td>host_disease_stat</td>
<td>0..1</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000032/">0000032</a></td>
<td>trophic_level</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000033/">0000033</a></td>
<td>propagation</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The type of reproduction from the parent stock. Values for this field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000003/">0000003</a></td>
<td>isol_growth_condt</td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collec_device</td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collec_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000016/">0000016</a></td>
<td>samp_mat_process</td>
<td>0..1</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000001/">0000001</a></td>
<td>samp_size</td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000111/">0000111</a></td>
<td>samp_vol_we_dna_ext</td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000037/">0000037</a></td>
<td>nucl_acid_ext</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000038/">0000038</a></td>
<td>nucl_acid_amp</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000039/">0000039</a></td>
<td>lib_size</td>
<td>0..1</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000040/">0000040</a></td>
<td>lib_reads_seqd</td>
<td>0..1</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000041/">0000041</a></td>
<td>lib_layout</td>
<td>0..1</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000042/">0000042</a></td>
<td>lib_vector</td>
<td>0..1</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000043/">0000043</a></td>
<td>lib_screen</td>
<td>0..1</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000048/">0000048</a></td>
<td>adapters</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
</tr>
<tr>
<td><a href="../0000050/">0000050</a></td>
<td>seq_meth</td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000053/">0000053</a></td>
<td>tax_ident</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000056/">0000056</a></td>
<td>assembly_qual</td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000057/">0000057</a></td>
<td>assembly_name</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
</tr>
<tr>
<td><a href="../0000058/">0000058</a></td>
<td>assembly_software</td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000059/">0000059</a></td>
<td>annot</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000060/">0000060</a></td>
<td>number_contig</td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000061/">0000061</a></td>
<td>feat_pred</td>
<td>0..1</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000062/">0000062</a></td>
<td>ref_db</td>
<td>0..1</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
</tr>
<tr>
<td><a href="../0000063/">0000063</a></td>
<td>sim_search_meth</td>
<td>0..1</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000064/">0000064</a></td>
<td>tax_class</td>
<td>0..1</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000069/">0000069</a></td>
<td>compl_score</td>
<td>0..1</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000070/">0000070</a></td>
<td>compl_software</td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>associated resource</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000090/">0000090</a></td>
<td>sop</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>0..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collect_device</td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000103/">0000103</a></td>
<td>organism_count</td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001335/">0001335</a></td>
<td>spikein_count</td>
<td>0..1</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) should also be provided (example: total prokaryotes; 3.5e7 cells per ml; qPCR).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000110/">0000110</a></td>
<td>samp_stor_temp</td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000116/">0000116</a></td>
<td>samp_stor_dur</td>
<td>0..1</td>
<td>Duration for which the sample was stored. Indicate the duration for which the sample was stored written in ISO 8601 format.</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0000325/">0000325</a></td>
<td>pool_dna_extracts</td>
<td>0..1</td>
<td>Indicate whether multiple DNA extractions were mixed. If the answer yes, the number of extracts that were pooled should be given</td>
<td>{boolean};{integer}</td>
</tr>
<tr>
<td><a href="../0000755/">0000755</a></td>
<td>samp_stor_loc</td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room. Indicate the location name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000859/">0000859</a></td>
<td>genetic_mod</td>
<td>0..1</td>
<td>Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000754/">0000754</a></td>
<td>perturbation</td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0001183/">0001183</a></td>
<td>coll_site_geo_feat</td>
<td>1..1</td>
<td>Text or terms that describe the geographic feature where the food sample was obtained by the researcher. This field accepts selected terms listed under the following ontologies: anthropogenic geographic feature (http://purl.obolibrary.org/obo/ENVO_00000002), for example agricultural fairground [ENVO:01000986]; garden [ENVO:00000011} or any of its subclasses; market [ENVO:01000987]; water well [ENVO:01000002]; or human construction (http://purl.obolibrary.org/obo/ENVO_00000070).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001184/">0001184</a></td>
<td>food_product_type</td>
<td>1..*</td>
<td>A food product type is a class of food products that is differentiated by its food composition (e.g., single- or multi-ingredient), processing and/or consumption characteristics. This does not include brand name products but it may include generic food dish categories. This field accepts terms under food product type (http://purl.obolibrary.org/obo/FOODON:03400361). For terms related to food product for an animal, consult food product for animal (http://purl.obolibrary.org/obo/FOODON_03309997). If the proper descriptor is not listed please use text to describe the food type. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001179/">0001179</a></td>
<td>IFSAC_category</td>
<td>1..*</td>
<td>The IFSAC food categorization scheme has five distinct levels to which foods can be assigned, depending upon the type of food. First, foods are assigned to one of four food groups (aquatic animals, land animals, plants, and other). Food groups include increasingly specific food categories; dairy, eggs, meat and poultry, and game are in the land animal food group, and the category meat and poultry is further subdivided into more specific categories of meat (beef, pork, other meat) and poultry (chicken, turkey, other poultry). Finally, foods are differentiated by differences in food processing (such as pasteurized fluid dairy products, unpasteurized fluid dairy products, pasteurized solid and semi-solid dairy products, and unpasteurized solid and semi-solid dairy products. An IFSAC food category chart is available from https://www.cdc.gov/foodsafety/ifsac/projects/food-categorization-scheme.html PMID: 28926300.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001185/">0001185</a></td>
<td>ferm_chem_add</td>
<td>0..*</td>
<td>Any chemicals that are added to the fermentation process to achieve the desired final product.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001186/">0001186</a></td>
<td>ferm_chem_add_perc</td>
<td>0..*</td>
<td>The amount of chemical added to the fermentation process.</td>
<td>{float} percentage</td>
</tr>
<tr>
<td><a href="../0001187/">0001187</a></td>
<td>ferm_headspace_oxy</td>
<td>0..1</td>
<td>The amount of headspace oxygen in a fermentation vessel.</td>
<td>{float} percentage</td>
</tr>
<tr>
<td><a href="../0001188/">0001188</a></td>
<td>ferm_medium</td>
<td>0..1</td>
<td>The growth medium used for the fermented food fermentation process, which supplies the required nutrients.  Usually this includes a carbon and nitrogen source, water, micronutrients and chemical additives.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001189/">0001189</a></td>
<td>ferm_pH</td>
<td>0..1</td>
<td>The pH of the fermented food fermentation process.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001190/">0001190</a></td>
<td>ferm_rel_humidity</td>
<td>0..1</td>
<td>The relative humidity of the fermented food fermentation process.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001191/">0001191</a></td>
<td>ferm_temp</td>
<td>0..1</td>
<td>The temperature of the fermented food fermentation process.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001192/">0001192</a></td>
<td>ferm_time</td>
<td>0..1</td>
<td>The time duration of the fermented food fermentation process.</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0001193/">0001193</a></td>
<td>ferm_vessel</td>
<td>0..1</td>
<td>The type of vessel used for containment of the fermentation.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001194/">0001194</a></td>
<td>bacterial_density</td>
<td>0..1</td>
<td>Number of bacteria in sample, as defined by bacteria density (http://purl.obolibrary.org/obo/GENEPIO_0000043).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001195/">0001195</a></td>
<td>cons_food_stor_dur</td>
<td>0..1</td>
<td>The storage duration of the food commodity by the consumer, prior to onset of illness or sample collection.  Indicate the timepoint written in ISO 8601 format.</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0001196/">0001196</a></td>
<td>cons_food_stor_temp</td>
<td>0..1</td>
<td>Temperature at which food commodity was stored by the consumer, prior to onset of illness or sample collection.</td>
<td>{float} {unit}</td>
</tr>
<tr>
<td><a href="../0001197/">0001197</a></td>
<td>cons_purch_date</td>
<td>0..1</td>
<td>The date a food product was purchased by consumer.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001198/">0001198</a></td>
<td>cons_qty_purchased</td>
<td>0..1</td>
<td>The quantity of food purchased by consumer.</td>
<td>{integer} {unit}</td>
</tr>
<tr>
<td><a href="../0001181/">0001181</a></td>
<td>cult_isol_date</td>
<td>0..1</td>
<td>A culture isolation date is a date-time entity marking the end of a process in which a sample yields a positive result for the target microbial analyte(s) in the form of an isolated colony or colonies.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001117/">0001117</a></td>
<td>cult_result</td>
<td>0..1</td>
<td>Any result of a bacterial culture experiment reported as a binary assessment such as positive/negative, active/inactive.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001118/">0001118</a></td>
<td>cult_result_org</td>
<td>0..*</td>
<td>Taxonomic information about the cultured organism(s).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001119/">0001119</a></td>
<td>cult_target</td>
<td>0..*</td>
<td>The target microbial analyte in terms of investigation scope. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001199/">0001199</a></td>
<td>dietary_claim_use</td>
<td>0..*</td>
<td>These descriptors are used either for foods intended for special dietary use as defined in 21 CFR 105 or for foods that have special characteristics indicated in the name or labeling. This field accepts terms listed under dietary claim or use (http://purl.obolibrary.org/obo/FOODON_03510023). Multiple terms can be separated by one or more pipes, but please consider limiting this list to the most prominent dietary claim or use.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001177/">0001177</a></td>
<td>enrichment_protocol</td>
<td>0..1</td>
<td>The microbiological workflow or protocol followed to test for the presence or enumeration of the target microbial analyte(s). Please provide a PubMed or DOI reference for published protocols.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001200/">0001200</a></td>
<td>food_additive</td>
<td>0..*</td>
<td>A substance or substances added to food to maintain or improve safety and freshness, to improve or maintain nutritional value, or improve taste, texture and appearance.  This field accepts terms listed under food additive (http://purl.obolibrary.org/obo/FOODON_03412972). Multiple terms can be separated by one or more pipes, but please consider limiting this list to the top 5 ingredients listed in order as on the food label.  See also, https://www.fda.gov/food/food-ingredients-packaging/overview-food-ingredients-additives-colors.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001201/">0001201</a></td>
<td>food_allergen_label</td>
<td>0..*</td>
<td>A label indication that the product contains a recognized allergen. This field accepts terms listed under dietary claim or use (http://purl.obolibrary.org/obo/FOODON_03510213).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001131/">0001131</a></td>
<td>food_contact_surf</td>
<td>0..*</td>
<td>The specific container or coating materials in direct contact with the food. Multiple values can be assigned.  This field accepts terms listed under food contact surface (http://purl.obolibrary.org/obo/FOODON_03500010).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001132/">0001132</a></td>
<td>food_contain_wrap</td>
<td>0..1</td>
<td>Type of container or wrapping defined by the main container material, the container form, and the material of the liner lids or ends. Also type of container or wrapping by form; prefer description by material first, then by form. This field accepts terms listed under food container or wrapping (http://purl.obolibrary.org/obo/FOODON_03490100).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001202/">0001202</a></td>
<td>food_cooking_proc</td>
<td>0..*</td>
<td>The transformation of raw food by the application of heat. This field accepts terms listed under food cooking (http://purl.obolibrary.org/obo/FOODON_03450002).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001203/">0001203</a></td>
<td>food_dis_point</td>
<td>0..*</td>
<td>A reference to a place on the Earth, by its name or by its geographical location that refers to a distribution point along the food chain. This field accepts terms listed under geographic location (http://purl.obolibrary.org/obo/GAZ_00000448). Reference: Adam Diamond, James Barham. Moving Food Along the Value Chain: Innovations in Regional Food Distribution. U.S. Dept. of Agriculture, Agricultural Marketing Service. Washington, DC. March 2012. http://dx.doi.org/10.9752/MS045.03-2012.</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0001205/">0001205</a></td>
<td>food_ingredient</td>
<td>0..*</td>
<td>In this field, please list individual ingredients for multi-component food [FOODON:00002501] and simple foods that is not captured in food_type.  Please use terms that are present in FoodOn.  Multiple terms can be separated by one or more pipes</td>
<td>, but please consider limiting this list to the top 5 ingredients listed in order as on the food label.  See also, https://www.fda.gov/food/food-ingredients-packaging/overview-food-ingredients-additives-colors.</td>
</tr>
<tr>
<td><a href="../0001206/">0001206</a></td>
<td>food_name_status</td>
<td>0..1</td>
<td>A datum indicating that use of a food product name is regulated in some legal jurisdiction. This field accepts terms listed under food product name legal status (http://purl.obolibrary.org/obo/FOODON_03530087).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001207/">0001207</a></td>
<td>food_origin</td>
<td>0..1</td>
<td>A reference to a place on the Earth, by its name or by its geographical location that describes the origin of the food commodity, either in terms of its cultivation or production. This field accepts terms listed under geographic location (http://purl.obolibrary.org/obo/GAZ_00000448).</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0001208/">0001208</a></td>
<td>food_pack_capacity</td>
<td>0..1</td>
<td>The maximum number of product units within a package</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001209/">0001209</a></td>
<td>food_pack_integrity</td>
<td>0..*</td>
<td>A term label and term id to describe the state of the packing material and text to explain the exact condition.  This field accepts terms listed under food packing medium integrity (http://purl.obolibrary.org/obo/FOODON_03530218).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001134/">0001134</a></td>
<td>food_pack_medium</td>
<td>0..*</td>
<td>The medium in which the food is packed for preservation and handling or the medium surrounding homemade foods, e.g., peaches cooked in sugar syrup. The packing medium may provide a controlled environment for the food. It may also serve to improve palatability and consumer appeal.  This includes edible packing media (e.g. fruit juice), gas other than air (e.g. carbon dioxide), vacuum packed, or packed with aerosol propellant. This field accepts terms under food packing medium (http://purl.obolibrary.org/obo/FOODON_03480020). Multiple terms may apply and can be separated by pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001135/">0001135</a></td>
<td>food_preserv_proc</td>
<td>0..*</td>
<td>The methods contributing to the prevention or retardation of microbial, enzymatic or oxidative spoilage and thus to the extension of shelf life. This field accepts terms listed under food preservation process (http://purl.obolibrary.org/obo/FOODON_03470107).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001210/">0001210</a></td>
<td>food_prior_contact</td>
<td>0..*</td>
<td>The material the food contacted (e.g., was processed in) prior to packaging. This field accepts terms listed under material of contact prior to food packaging (http://purl.obolibrary.org/obo/FOODON_03530077). If the proper descriptor is not listed please use text to describe the material of contact prior to food packaging.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001211/">0001211</a></td>
<td>food_prod</td>
<td>0..*</td>
<td>Descriptors of the food production system or of the agricultural environment and growing conditions related to the farm production system, such as wild caught, organic, free-range, industrial, dairy, beef,  domestic or cultivated food production. This field accepts terms listed under food production (http://purl.obolibrary.org/obo/FOODON_03530206). Multiple terms may apply and can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001212/">0001212</a></td>
<td>food_prod_synonym</td>
<td>0..*</td>
<td>Other names by which the food product is known by (e.g., regional or non-English names).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001213/">0001213</a></td>
<td>food_product_qual</td>
<td>0..1</td>
<td>Descriptors for describing food visually or via other senses, which is useful for tasks like food inspection where little prior knowledge of how the food came to be is available. Some terms like "food (frozen)" are both a quality descriptor and the output of a process. This field accepts terms listed under food product by quality (http://purl.obolibrary.org/obo/FOODON_00002454).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001178/">0001178</a></td>
<td>food_quality_date</td>
<td>0..1</td>
<td>The date recommended for the use of the product while at peak quality, this date is not a reflection of safety unless used on infant formula this date is not a reflection of safety and is typically labeled on a food product as "best if used by," best by," "use by," or "freeze by."</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001139/">0001139</a></td>
<td>food_source</td>
<td>0..1</td>
<td>Type of plant or animal from which the food product or its major ingredient is derived or a chemical food source [FDA CFSAN 1995].</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001214/">0001214</a></td>
<td>food_trace_list</td>
<td>0..1</td>
<td>The FDA is proposing to establish additional traceability recordkeeping requirements (beyond what is already required in existing regulations) for persons who manufacture, process, pack, or hold foods the Agency has designated for inclusion on the Food Traceability List. The Food Traceability List (FTL) identifies the foods for which the additional traceability records described in the proposed rule would be required. The term “Food Traceability List” (FTL) refers not only to the foods specifically listed (https://www.fda.gov/media/142303/download), but also to any foods that contain listed foods as ingredients.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001137/">0001137</a></td>
<td>food_trav_mode</td>
<td>0..*</td>
<td>A descriptor for the method of movement of food commodity along the food distribution system.  This field accepts terms listed under travel mode (http://purl.obolibrary.org/obo/GENEPIO_0001064). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001138/">0001138</a></td>
<td>food_trav_vehic</td>
<td>0..*</td>
<td>A descriptor for the mobile machine which is used to transport food commodities along the food distribution system.  This field accepts terms listed under vehicle (http://purl.obolibrary.org/obo/ENVO_01000604). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001140/">0001140</a></td>
<td>food_treat_proc</td>
<td>0..*</td>
<td>Used to specifically characterize a food product based on the treatment or processes applied to the product or any indexed ingredient. The processes include adding, substituting or removing components or modifying the food or component, e.g., through fermentation. Multiple values can be assigned. This fields accepts terms listed under food treatment process (http://purl.obolibrary.org/obo/FOODON_03460111).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001215/">0001215</a></td>
<td>HACCP_term</td>
<td>0..*</td>
<td>Hazard Analysis Critical Control Points (HACCP) food safety terms; This field accepts terms listed under HACCP guide food safety term (http://purl.obolibrary.org/obo/FOODON_03530221).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001144/">0001144</a></td>
<td>intended_consumer</td>
<td>0..*</td>
<td>Food consumer type, human or animal, for which the food product is produced and marketed. This field accepts terms listed under food consumer group (http://purl.obolibrary.org/obo/FOODON_03510136) or NCBI taxid.</td>
<td>{integer}</td>
</tr>
<tr>
<td><a href="../0001145/">0001145</a></td>
<td>library_prep_kit</td>
<td>0..1</td>
<td>Packaged kits (containing adapters, indexes, enzymes, buffers etc.), tailored for specific sequencing workflows, which allow the simplified preparation of sequencing-ready libraries for small genomes, amplicons, and plasmids.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001147/">0001147</a></td>
<td>lot_number</td>
<td>0..*</td>
<td>A distinctive alpha-numeric identification code assigned by the manufacturer or distributor to a specific quantity of manufactured material or product within a batch. Synonym: Batch Number.  The submitter should provide lot number of the item followed by the item name for which the lot number was provided.</td>
<td>{integer}, {text}</td>
</tr>
<tr>
<td><a href="../0001216/">0001216</a></td>
<td>microb_cult_med</td>
<td>0..1</td>
<td>A culture medium used to select for, grow, and maintain prokaryotic microorganisms. Can be in either liquid (broth) or solidified (e.g. with agar) forms. This field accepts terms listed under microbiological culture medium (http://purl.obolibrary.org/obo/MICRO_0000067). If the proper descriptor is not listed please use text to describe the culture medium.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001217/">0001217</a></td>
<td>microb_start</td>
<td>0..1</td>
<td>Any type of microorganisms used in food production.  This field accepts terms listed under live organisms for food production (http://purl.obolibrary.org/obo/FOODON_0344453).</td>
<td>{term label} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001218/">0001218</a></td>
<td>microb_start_count</td>
<td>0..1</td>
<td>Total cell count of starter culture per gram, volume or area of sample and the method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) should also be provided. (example : total prokaryotes; 3.5e7 cells per ml; qPCR).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001219/">0001219</a></td>
<td>microb_start_inoc</td>
<td>0..1</td>
<td>The amount of starter culture used to inoculate a new batch.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001220/">0001220</a></td>
<td>microb_start_prep</td>
<td>0..1</td>
<td>Information about the protocol or method used to prepare the starter inoculum.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001221/">0001221</a></td>
<td>microb_start_source</td>
<td>0..1</td>
<td>The source from which the microbial starter culture was sourced.  If commercially supplied, list supplier.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001222/">0001222</a></td>
<td>microb_start_taxID</td>
<td>0..1</td>
<td>Please include Genus species and strain ID, if known of microorganisms used in food production. For complex communities, pipes can be used to separate two or more microbes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001223/">0001223</a></td>
<td>nucl_acid_ext_kit</td>
<td>0..*</td>
<td>The name of the extraction kit used to recover the nucleic acid fraction of an input material is performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001224/">0001224</a></td>
<td>num_samp_collect</td>
<td>0..1</td>
<td>The number of samples collected during the current sampling event.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001149/">0001149</a></td>
<td>part_plant_animal</td>
<td>0..*</td>
<td>The anatomical part of the organism being involved in food production or consumption; e.g., a carrot is the root of the plant (root vegetable). This field accepts terms listed under part of plant or animal (http://purl.obolibrary.org/obo/FOODON_03420116).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001152/">0001152</a></td>
<td>repository_name</td>
<td>0..*</td>
<td>The name of the institution where the sample or DNA extract is held or "sample not available" if the sample was used in its entirety for analysis or otherwise not retained.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>sample_collec_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001153/">0001153</a></td>
<td>samp_pooling</td>
<td>0..*</td>
<td>Physical combination of several instances of like material, e.g. RNA extracted from samples or dishes of cell cultures into one big aliquot of cells. Please provide a short description of the samples that were pooled.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001226/">0001226</a></td>
<td>samp_rep_biol</td>
<td>0..1</td>
<td>Measurements of biologically distinct samples that show biological variation.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001227/">0001227</a></td>
<td>samp_rep_tech</td>
<td>0..1</td>
<td>Repeated measurements of the same sample that show independent measures of the noise associated with the equipment and the protocols.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001154/">0001154</a></td>
<td>samp_source_mat_cat</td>
<td>0..1</td>
<td>This is the scientific role or category that the subject organism or material has with respect to an investigation.  This field accepts terms listed under specimen source material category (http://purl.obolibrary.org/obo/GENEPIO_0001237 or http://purl.obolibrary.org/obo/OBI_0100051).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001228/">0001228</a></td>
<td>samp_stor_device</td>
<td>0..1</td>
<td>The container used to store the  sample. This field accepts terms listed under container (http://purl.obolibrary.org/obo/NCIT_C43186). If the proper descriptor is not listed please use text to describe the storage device.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001229/">0001229</a></td>
<td>samp_stor_media</td>
<td>0..1</td>
<td>The liquid that is added to the sample collection device prior to sampling. If the sample is pre-hydrated, indicate the liquid media the sample is pre-hydrated with for storage purposes. This field accepts terms listed under microbiological culture medium (http://purl.obolibrary.org/obo/MICRO_0000067). If the proper descriptor is not listed please use text to describe the sample storage media.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001230/">0001230</a></td>
<td>samp_transport_cont</td>
<td>0..1</td>
<td>Conatiner in which the sample was stored during transport. Indicate the location name.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001231/">0001231</a></td>
<td>samp_transport_dur</td>
<td>0..1</td>
<td>The duration of time from when the sample was collected until processed. Indicate the duration for which the sample was stored written in ISO 8601 format.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001232/">0001232</a></td>
<td>samp_transport_temp</td>
<td>0..1</td>
<td>Temperature at which sample was transported, e.g. -20 or 4 degree Celsius.</td>
<td>{float} {unit} {text}</td>
</tr>
<tr>
<td><a href="../0001151/">0001151</a></td>
<td>samp_purpose</td>
<td>0..1</td>
<td>The reason that the sample was collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001155/">0001155</a></td>
<td>sequencing_kit</td>
<td>0..1</td>
<td>Pre-filled, ready-to-use reagent cartridges. Used to produce improved chemistry, cluster density and read length as well as improve quality (Q) scores. Reagent components are encoded to interact with the sequencing system to validate compatibility with user-defined applications. Indicate name of the sequencing kit.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001156/">0001156</a></td>
<td>sequencing_location</td>
<td>0..1</td>
<td>The location the sequencing run was performed. Indicate the name of the lab or core facility where samples were sequenced.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001157/">0001157</a></td>
<td>serovar_or_serotype</td>
<td>0..*</td>
<td>A characterization of a cell or microorganism based on the antigenic properties of the molecules on its surface. Indicate the name of a serovar or serotype of interest. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001235/">0001235</a></td>
<td>spikein_AMR</td>
<td>0..*</td>
<td>Qualitative description of a microbial response to antimicrobial agents. Bacteria may be susceptible or resistant to a broad range of antibiotic drugs or drug classes, with several intermediate states or phases. This field accepts terms under antimicrobial phenotype (http://purl.obolibrary.org/obo/ARO_3004299).</td>
<td>{float} {unit};{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001171/">0001171</a></td>
<td>spikein_antibiotic</td>
<td>0..*</td>
<td>Antimicrobials used in research study to assess effects of exposure on microbiome of a specific site.  Please list antimicrobial, common name and/or class and concentration used for spike-in.</td>
<td>{text} {integer}</td>
</tr>
<tr>
<td><a href="../0001169/">0001169</a></td>
<td>spikein_growth_med</td>
<td>0..*</td>
<td>A liquid or gel containing nutrients, salts, and other factors formulated to support the growth of microorganisms, cells, or plants (National Cancer Institute Thesaurus).  A growth medium is a culture medium which has the disposition to encourage growth of particular bacteria to the exclusion of others in the same growth environment.  In this case, list the culture medium used to propagate the spike-in bacteria during preparation of spike-in inoculum. This field accepts terms listed under microbiological culture medium (http://purl.obolibrary.org/obo/MICRO_0000067). If the proper descriptor is not listed please use text to describe the spike in growth media.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001172/">0001172</a></td>
<td>spikein_metal</td>
<td>0..*</td>
<td>Heavy metals used in research study to assess effects of exposure on microbiome of a specific site.  Please list heavy metals and concentration used for spike-in.</td>
<td>{text} {integer}</td>
</tr>
<tr>
<td><a href="../0001167/">0001167</a></td>
<td>spikein_org</td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s). This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001168/">0001168</a></td>
<td>spikein_serovar</td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s) at the serovar or serotype level. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001170/">0001170</a></td>
<td>spikein_strain</td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s) at the strain level. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001236/">0001236</a></td>
<td>study_design</td>
<td>0..*</td>
<td>A plan specification comprised of protocols (which may specify how and what kinds of data will be gathered) that are executed as part of an investigation and is realized during a study design execution. This field accepts terms under study design (http://purl.obolibrary.org/obo/OBI_0500000). If the proper descriptor is not listed please use text to describe the study design. Multiple terms can be separated by pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001237/">0001237</a></td>
<td>study_inc_dur</td>
<td>0..1</td>
<td>Sample incubation duration if unpublished or unvalidated method is used. Indicate the timepoint written in ISO 8601 format.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001238/">0001238</a></td>
<td>study_inc_temp</td>
<td>0..1</td>
<td>Sample incubation temperature if unpublished or unvalidated method is used.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001239/">0001239</a></td>
<td>study_timecourse</td>
<td>0..1</td>
<td>For time-course research studies involving samples of the food commodity, indicate the total duration of the time-course study.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001240/">0001240</a></td>
<td>study_tmnt</td>
<td>0..*</td>
<td>A process in which the act is intended to modify or alter some other material entity.  From the study design, each treatment is comprised of one level of one or multiple factors. This field accepts terms listed under treatment (http://purl.obolibrary.org/obo/MCO_0000866). If the proper descriptor is not listed please use text to describe the study treatment. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001173/">0001173</a></td>
<td>timepoint</td>
<td>0..1</td>
<td>Time point at which a sample or observation is made or taken from a biomaterial as measured from some reference point. Indicate the timepoint written in ISO 8601 format.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
