
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/AgricultureMIUVIG/">
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.3.0, mkdocs-material-8.3.7">
    
    
      
        <title>AgricultureMIUVIG - MIxS</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.4a0965b7.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
      
      
    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="none" data-md-color-accent="none">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#agriculturemiuvig" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="MIxS" class="md-header__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            MIxS
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              AgricultureMIUVIG
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="MIxS" class="md-nav__button md-logo" aria-label="MIxS" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"/></svg>

    </a>
    MIxS
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/GenomicsStandardsConsortium/mixs/" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.1.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Reference
      </a>
    </li>
  

    
      
      
      

  
  
  
    
    <li class="md-nav__item md-nav__item--nested">
      
      
        <input class="md-nav__toggle md-toggle" data-md-toggle="__nav_2" type="checkbox" id="__nav_2" >
      
      
      
      
        <label class="md-nav__link" for="__nav_2">
          Introduction
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <nav class="md-nav" aria-label="Introduction" data-md-level="1">
        <label class="md-nav__title" for="__nav_2">
          <span class="md-nav__icon md-icon"></span>
          Introduction
        </label>
        <ul class="md-nav__list" data-md-scrollfix>
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/background/" class="md-nav__link">
        Background
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/citation/" class="md-nav__link">
        Cite
      </a>
    </li>
  

            
          
            
              
  
  
  
    <li class="md-nav__item">
      <a href="../introduction/contents/" class="md-nav__link">
        Contents
      </a>
    </li>
  

            
          
        </ul>
      </nav>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#inheritance" class="md-nav__link">
    Inheritance
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    Slots
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          <div class="md-content" data-md-component="content">
            <article class="md-content__inner md-typeset">
              
                
  <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/AgricultureMIUVIG.md" title="Edit this page" class="md-content__button md-icon">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>



<h1 id="agriculturemiuvig">AgricultureMIUVIG</h1>
<p>Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package agriculture</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/AgricultureMIUVIG">mixs.vocab:AgricultureMIUVIG</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Agriculture/">Agriculture</a><ul>
<li><strong>AgricultureMIUVIG</strong> [ MIUVIG]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../0001107/">0001107</a></td>
<td>samp_name</td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001320/">0001320</a></td>
<td>samp_taxon_id</td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
</tr>
<tr>
<td><a href="../0000092/">0000092</a></td>
<td>project_name</td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000008/">0000008</a></td>
<td>experimental_factor</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000009/">0000009</a></td>
<td>lat_lon</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
</tr>
<tr>
<td><a href="../0000010/">0000010</a></td>
<td>geo_loc_name</td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
</tr>
<tr>
<td><a href="../0000011/">0000011</a></td>
<td>collection_date</td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001321/">0001321</a></td>
<td>neg_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001322/">0001322</a></td>
<td>pos_cont_type</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
</tr>
<tr>
<td><a href="../0000012/">0000012</a></td>
<td>env_broad_scale</td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000013/">0000013</a></td>
<td>env_local_scale</td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000014/">0000014</a></td>
<td>env_medium</td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000024/">0000024</a></td>
<td>estimated_size</td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
</tr>
<tr>
<td><a href="../0000025/">0000025</a></td>
<td>ref_biomaterial</td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000026/">0000026</a></td>
<td>source_mat_id</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000027/">0000027</a></td>
<td>pathogenicity</td>
<td>0..1</td>
<td>To what is the entity pathogenic</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000028/">0000028</a></td>
<td>biotic_relationship</td>
<td>0..1</td>
<td>Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000029/">0000029</a></td>
<td>specific_host</td>
<td>0..1</td>
<td>Report the host's taxonomic name and/or NCBI taxonomy ID.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000030/">0000030</a></td>
<td>host_spec_range</td>
<td>0..1</td>
<td>The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000031/">0000031</a></td>
<td>host_disease_stat</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collec_device</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collec_method</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000016/">0000016</a></td>
<td>samp_mat_process</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000017/">0000017</a></td>
<td>size_frac</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Filtering pore size used in sample preparation</td>
<td>{float}-{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000001/">0000001</a></td>
<td>samp_size</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000111/">0000111</a></td>
<td>samp_vol_we_dna_ext</td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000035/">0000035</a></td>
<td>source_uvig</td>
<td>1..1</td>
<td>Type of dataset from which the UViG was obtained</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000036/">0000036</a></td>
<td>virus_enrich_appr</td>
<td>1..1</td>
<td>List of approaches used to enrich the sample for viruses, if any</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000037/">0000037</a></td>
<td>nucl_acid_ext</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000038/">0000038</a></td>
<td>nucl_acid_amp</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000039/">0000039</a></td>
<td>lib_size</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000040/">0000040</a></td>
<td>lib_reads_seqd</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000041/">0000041</a></td>
<td>lib_layout</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000042/">0000042</a></td>
<td>lib_vector</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000043/">0000043</a></td>
<td>lib_screen</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000047/">0000047</a></td>
<td>mid</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</td>
<td>{dna}</td>
</tr>
<tr>
<td><a href="../0000048/">0000048</a></td>
<td>adapters</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
</tr>
<tr>
<td><a href="../0000050/">0000050</a></td>
<td>seq_meth</td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000053/">0000053</a></td>
<td>tax_ident</td>
<td>0..1</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000056/">0000056</a></td>
<td>assembly_qual</td>
<td>1..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000057/">0000057</a></td>
<td>assembly_name</td>
<td>1..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
</tr>
<tr>
<td><a href="../0000058/">0000058</a></td>
<td>assembly_software</td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000059/">0000059</a></td>
<td>annot</td>
<td>0..1</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000060/">0000060</a></td>
<td>number_contig</td>
<td>1..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000061/">0000061</a></td>
<td>feat_pred</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000062/">0000062</a></td>
<td>ref_db</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
</tr>
<tr>
<td><a href="../0000063/">0000063</a></td>
<td>sim_search_meth</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000064/">0000064</a></td>
<td>tax_class</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000067/">0000067</a></td>
<td>trnas</td>
<td>0..1</td>
<td>The total number of tRNAs identified from the SAG or MAG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000068/">0000068</a></td>
<td>trna_ext_software</td>
<td>0..1</td>
<td>Tools used for tRNA identification</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000069/">0000069</a></td>
<td>compl_score</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000070/">0000070</a></td>
<td>compl_software</td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
</tr>
<tr>
<td><a href="../0000071/">0000071</a></td>
<td>compl_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000075/">0000075</a></td>
<td>sort_tech</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to sort/isolate cells or particles of interest</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000076/">0000076</a></td>
<td>single_cell_lysis_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to free DNA from interior of the cell(s) or particle(s)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000054/">0000054</a></td>
<td>single_cell_lysis_prot</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name of the kit or standard protocol used for cell(s) or particle(s) lysis</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000055/">0000055</a></td>
<td>wga_amp_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Method used to amplify genomic DNA in preparation for sequencing</td>
<td>[pcr based</td>
</tr>
<tr>
<td><a href="../0000006/">0000006</a></td>
<td>wga_amp_kit</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Kit used to amplify genomic DNA in preparation for sequencing</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000077/">0000077</a></td>
<td>bin_param</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The parameters that have been applied during the extraction of genomes from metagenomic datasets</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000078/">0000078</a></td>
<td>bin_software</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</td>
<td>{software};{version}{PID}</td>
</tr>
<tr>
<td><a href="../0000079/">0000079</a></td>
<td>reassembly_bin</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</td>
<td>{boolean}</td>
</tr>
<tr>
<td><a href="../0000080/">0000080</a></td>
<td>mag_cov_software</td>
<td>0..1</td>
<td>Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000081/">0000081</a></td>
<td>vir_ident_software</td>
<td>1..1</td>
<td>Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000082/">0000082</a></td>
<td>pred_genome_type</td>
<td>1..1</td>
<td>Type of genome predicted for the UViG</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000083/">0000083</a></td>
<td>pred_genome_struc</td>
<td>1..1</td>
<td>Expected structure of the viral genome</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000084/">0000084</a></td>
<td>detec_type</td>
<td>1..1</td>
<td>Type of UViG detection</td>
<td>[independent sequence (UViG)</td>
</tr>
<tr>
<td><a href="../0000085/">0000085</a></td>
<td>otu_class_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Cutoffs and approach used when clustering “species-level” OTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside OTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis</td>
<td>{ANI cutoff};{AF cutoff};{clustering method}</td>
</tr>
<tr>
<td><a href="../0000086/">0000086</a></td>
<td>otu_seq_comp_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool and thresholds used to compare sequences when computing "species-level" OTUs</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000087/">0000087</a></td>
<td>otu_db</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" OTUs, if any</td>
<td>{database};{version}</td>
</tr>
<tr>
<td><a href="../0000088/">0000088</a></td>
<td>host_pred_appr</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool or approach used for host prediction</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000089/">0000089</a></td>
<td>host_pred_est_acc</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>associated resource</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000090/">0000090</a></td>
<td>sop</td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000094/">0000094</a></td>
<td>alt</td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000093/">0000093</a></td>
<td>elev</td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000113/">0000113</a></td>
<td>temp</td>
<td>1..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001057/">0001057</a></td>
<td>plant_growth_med</td>
<td>0..1</td>
<td>Specification of the media for growing the plants or tissue cultured samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture medium, in vitro liquid culture medium. Recommended value is a specific value from EO:plant growth medium (follow this link for terms http://purl.obolibrary.org/obo/EO_0007147) or other controlled vocabulary</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001082/">0001082</a></td>
<td>soil_horizon</td>
<td>0..1</td>
<td>Specific layer in the land area which measures parallel to the soil surface and possesses physical characteristics which differ from the layers above and beneath</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001296/">0001296</a></td>
<td>photosynt_activ</td>
<td>0..*</td>
<td>Measurement of photosythetic activity (i.e. leaf gas exchange / chlorophyll fluorescence emissions / reflectance / transpiration) Please also include the term method term detailing the method of activity measurement.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001336/">0001336</a></td>
<td>photosynt_activ_meth</td>
<td>0..*</td>
<td>Reference or method used in measurement of photosythetic activity</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001225/">0001225</a></td>
<td>samp_collect_method</td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001177/">0001177</a></td>
<td>enrichment_protocol</td>
<td>0..1</td>
<td>The microbiological workflow or protocol followed to test for the presence or enumeration of the target microbial analyte(s). Please provide a PubMed or DOI reference for published protocols.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001145/">0001145</a></td>
<td>library_prep_kit</td>
<td>0..1</td>
<td>Packaged kits (containing adapters, indexes, enzymes, buffers etc.), tailored for specific sequencing workflows, which allow the simplified preparation of sequencing-ready libraries for small genomes, amplicons, and plasmids.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001156/">0001156</a></td>
<td>sequencing_location</td>
<td>0..1</td>
<td>The location the sequencing run was performed. Indicate the name of the lab or core facility where samples were sequenced.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001163/">0001163</a></td>
<td>soil_temp</td>
<td>0..*</td>
<td>Temperature of soil at the time of sampling.</td>
<td>{float}{unit}</td>
</tr>
<tr>
<td><a href="../0001160/">0001160</a></td>
<td>soil_pH</td>
<td>0..1</td>
<td>The pH of soil at time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001158/">0001158</a></td>
<td>soil_conductivity</td>
<td>0..1</td>
<td>Conductivity of soil at time of sampling.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001161/">0001161</a></td>
<td>rel_location</td>
<td>0..1</td>
<td>Location of sampled soil to other parts of the farm e.g. under crop plant, near irrigation ditch, from the dirt road.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001159/">0001159</a></td>
<td>soil_cover</td>
<td>0..1</td>
<td>Description of the material covering the sampled soil. This field accepts terms under ENVO:00010483, environmental material.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000211/">0000211</a></td>
<td>porosity</td>
<td>0..1</td>
<td>Porosity of deposited sediment is volume of voids divided by the total volume of sample</td>
<td>{float} - {float} {unit}</td>
</tr>
<tr>
<td><a href="../0000335/">0000335</a></td>
<td>texture</td>
<td>0..1</td>
<td>The relative proportion of different grain sizes of mineral particles in a soil, as described using a standard system; express as % sand (50 um to 2 mm), silt (2 um to 50 um), and clay (&lt;2 um) with textural name (e.g., silty clay loam) optional.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000336/">0000336</a></td>
<td>texture_meth</td>
<td>0..1</td>
<td>Reference or method used in determining soil texture.</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001298/">0001298</a></td>
<td>host_symbiont</td>
<td>0..*</td>
<td>The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000819/">0000819</a></td>
<td>pres_animal_insect</td>
<td>0..1</td>
<td>The type and number of animals or insects present in the sampling space.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001111/">0001111</a></td>
<td>plant_water_method</td>
<td>0..1</td>
<td>Description of the equipment or method used to distribute water to crops. This field accepts termed listed under irrigation process (http://purl.obolibrary.org/obo/AGRO_00000006). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001115/">0001115</a></td>
<td>anim_water_method</td>
<td>0..*</td>
<td>Description of the equipment or method used to distribute water to livestock. This field accepts termed listed under water delivery equipment (http://opendata.inra.fr/EOL/EOL_0001653). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001110/">0001110</a></td>
<td>farm_water_source</td>
<td>0..*</td>
<td>Source of water used on the farm for irrigation of crops or watering of livestock.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001176/">0001176</a></td>
<td>water_source_shared</td>
<td>0..*</td>
<td>Other users sharing access to the same water source. Multiple terms can be separated by one or more pipes.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001175/">0001175</a></td>
<td>water_pH</td>
<td>0..1</td>
<td>pH measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000829/">0000829</a></td>
<td>season</td>
<td>0..1</td>
<td>The season when sampling occurred. Any of the four periods into which the year is divided by the equinoxes and solstices. This field accepts terms listed under season (http://purl.obolibrary.org/obo/NCIT_C94729).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000112/">0000112</a></td>
<td>solar_irradiance</td>
<td>0..1</td>
<td>The amount of solar energy that arrives at a specific area of a surface during a specific time interval</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001116/">0001116</a></td>
<td>crop_yield</td>
<td>0..1</td>
<td>Amount of crop produced per unit or area of land.</td>
<td>{float}{unit}</td>
</tr>
<tr>
<td><a href="../0001148/">0001148</a></td>
<td>season_humidity</td>
<td>0..1</td>
<td>Average humidity of the region throughout the growing season.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000100/">0000100</a></td>
<td>humidity</td>
<td>0..1</td>
<td>Amount of water vapour in the air, at the time of sampling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001121/">0001121</a></td>
<td>adjacent_environment</td>
<td>0..*</td>
<td>Description of the environmental system or features that are adjacent to the sampling site. This field accepts terms under ecosystem (http://purl.obolibrary.org/obo/ENVO_01001110) and human construction (http://purl.obolibrary.org/obo/ENVO_00000070). Multiple terms can be separated by pipes.</td>
<td>{termLabel}{[termID]}</td>
</tr>
<tr>
<td><a href="../0000751/">0000751</a></td>
<td>chem_administration</td>
<td>1..*</td>
<td>List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</td>
<td>{termLabel} {[termID]};{timestamp}</td>
</tr>
<tr>
<td><a href="../0001211/">0001211</a></td>
<td>food_prod</td>
<td>0..*</td>
<td>Descriptors of the food production system or of the agricultural environment and growing conditions related to the farm production system, such as wild caught, organic, free-range, industrial, dairy, beef,  domestic or cultivated food production. This field accepts terms listed under food production (http://purl.obolibrary.org/obo/FOODON_03530206). Multiple terms may apply and can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001147/">0001147</a></td>
<td>lot_number</td>
<td>0..*</td>
<td>A distinctive alpha-numeric identification code assigned by the manufacturer or distributor to a specific quantity of manufactured material or product within a batch. Synonym: Batch Number.  The submitter should provide lot number of the item followed by the item name for which the lot number was provided.</td>
<td>{integer}, {text}</td>
</tr>
<tr>
<td><a href="../0001127/">0001127</a></td>
<td>fertilizer_admin</td>
<td>0..1</td>
<td>Type of fertilizer or amendment added to the soil or water for the purpose of improving substrate health and quality for plant growth. This field accepts terms listed under agronomic fertilizer (http://purl.obolibrary.org/obo/AGRO_00002062). Multiple terms may apply and can be separated by pipes, listing in reverse chronological order.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000110/">0000110</a></td>
<td>samp_stor_temp</td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001137/">0001137</a></td>
<td>food_trav_mode</td>
<td>0..*</td>
<td>A descriptor for the method of movement of food commodity along the food distribution system.  This field accepts terms listed under travel mode (http://purl.obolibrary.org/obo/GENEPIO_0001064). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001138/">0001138</a></td>
<td>food_trav_vehic</td>
<td>0..*</td>
<td>A descriptor for the mobile machine which is used to transport food commodities along the food distribution system.  This field accepts terms listed under vehicle (http://purl.obolibrary.org/obo/ENVO_01000604). If the proper descrptor is not listed please use text to describe the mode of travel. Multiple terms can be separated by one or more pipes.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001124/">0001124</a></td>
<td>farm_equip_san</td>
<td>0..*</td>
<td>Method used to sanitize growing and harvesting equipment. This can including type and concentration of sanitizing solution.  Multiple terms can be separated by one or more pipes.</td>
<td>{text} {float} {unit}</td>
</tr>
<tr>
<td><a href="../0001126/">0001126</a></td>
<td>farm_equip</td>
<td>0..*</td>
<td>List of equipment used for planting, fertilization, harvesting, irrigation, land levelling, residue management, weeding or transplanting during the growing season.  This field accepts terms listed under agricultural implement (http://purl.obolibrary.org/obo/AGRO_00000416). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001123/">0001123</a></td>
<td>farm_equip_shared</td>
<td>0..*</td>
<td>List of planting, growing or harvesting equipment shared with other farms. This field accepts terms listed under agricultural implement (http://purl.obolibrary.org/obo/AGRO_00000416). Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0001133/">0001133</a></td>
<td>food_harvest_proc</td>
<td>0..*</td>
<td>A harvesting process is a process which takes in some food material from an individual or community of plant or animal organisms in a given context and time, and outputs a precursor or consumable food product. This may include a part of an organism or the whole, and may involve killing the organism.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001060/">0001060</a></td>
<td>plant_struc</td>
<td>0..1</td>
<td>Name of plant structure the sample was obtained from; for Plant Ontology (PO) (v releases/2017-12-14) terms, see http://purl.bioontology.org/ontology/PO, e.g. petiole epidermis (PO_0000051). If an individual flower is sampled, the sex of it can be recorded here.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000257/">0000257</a></td>
<td>host_dry_mass</td>
<td>0..1</td>
<td>Measurement of dry mass</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000247/">0000247</a></td>
<td>ances_data</td>
<td>0..1</td>
<td>Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000859/">0000859</a></td>
<td>genetic_mod</td>
<td>0..1</td>
<td>Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001184/">0001184</a></td>
<td>Food_Product_type</td>
<td>0..1</td>
<td>new food product categories and food products themselves</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001139/">0001139</a></td>
<td>Food_source</td>
<td>1..1</td>
<td>Type of plant or animal from which the food product or its major ingredient is derived or a chemical food source [FDA CFSAN 1995].</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0001170/">0001170</a></td>
<td>spikein_strain</td>
<td>0..*</td>
<td>Taxonomic information about the spike-in organism(s) at the strain level. This field accepts terms under organism (http://purl.obolibrary.org/obo/NCIT_C14250). This field also accepts identification numbers from NCBI under https://www.ncbi.nlm.nih.gov/taxonomy. Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000103/">0000103</a></td>
<td>organism_count</td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000735/">0000735</a></td>
<td>size_frac_low</td>
<td>0..1</td>
<td>Refers to the mesh/pore size used to pre-filter/pre-sort the sample. Materials larger than the size threshold are excluded from the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000736/">0000736</a></td>
<td>size_frac_up</td>
<td>0..1</td>
<td>Refers to the mesh/pore size used to retain the sample. Materials smaller than the size threshold are excluded from the sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001181/">0001181</a></td>
<td>cult_isol_date</td>
<td>0..1</td>
<td>A culture isolation date is a date-time entity marking the end of a process in which a sample yields a positive result for the target microbial analyte(s) in the form of an isolated colony or colonies.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001153/">0001153</a></td>
<td>samp_pooling</td>
<td>0..*</td>
<td>Physical combination of several instances of like material, e.g. RNA extracted from samples or dishes of cell cultures into one big aliquot of cells. Please provide a short description of the samples that were pooled.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000578/">0000578</a></td>
<td>root_med_macronutr</td>
<td>0..1</td>
<td>Measurement of the culture rooting medium macronutrients (N,P, K, Ca, Mg, S); e.g. KH2PO4 (170¬†mg/L).</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0000577/">0000577</a></td>
<td>root_med_carbon</td>
<td>0..1</td>
<td>Source of organic carbon in the culture rooting medium; e.g. sucrose.</td>
<td>{text};{float} {unit}</td>
</tr>
<tr>
<td><a href="../0001062/">0001062</a></td>
<td>root_med_ph</td>
<td>0..1</td>
<td>pH measurement of the culture rooting medium; e.g. 5.5.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000018/">0000018</a></td>
<td>soil_depth</td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. For sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000590/">0000590</a></td>
<td>water_temp_regm</td>
<td>0..*</td>
<td>Information about treatment involving an exposure to water with varying degree of temperature, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</td>
<td>{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000591/">0000591</a></td>
<td>watering_regm</td>
<td>0..*</td>
<td>Information about treatment involving an exposure to watering frequencies, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</td>
<td>{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0001069/">0001069</a></td>
<td>standing_water_regm</td>
<td>0..*</td>
<td>Treatment involving an exposure to standing water during a plant's life span, types can be flood water or standing water, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000558/">0000558</a></td>
<td>gaseous_environment</td>
<td>0..*</td>
<td>Use of conditions with differing gaseous environments; should include the name of gaseous compound, amount administered, treatment duration, interval and total experimental duration; can include multiple gaseous environment regimens</td>
<td>{text};{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000557/">0000557</a></td>
<td>fungicide_regm</td>
<td>0..*</td>
<td>Information about treatment involving use of fungicides; should include the name of fungicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple fungicide regimens</td>
<td>{text};{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0001040/">0001040</a></td>
<td>climate_environment</td>
<td>0..*</td>
<td>Treatment involving an exposure to a particular climate; treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple climates</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000561/">0000561</a></td>
<td>herbicide_regm</td>
<td>0..*</td>
<td>Information about treatment involving use of herbicides; information about treatment involving use of growth hormones; should include the name of herbicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</td>
<td>{text};{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000571/">0000571</a></td>
<td>non_mineral_nutr_regm</td>
<td>0..*</td>
<td>Information about treatment involving the exposure of plant to non-mineral nutrient such as oxygen, hydrogen or carbon; should include the name of non-mineral nutrient, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple non-mineral nutrient regimens</td>
<td>{text};{float} {unit};{period};{interval};{period}</td>
</tr>
<tr>
<td><a href="../0000573/">0000573</a></td>
<td>pesticide_regm</td>
<td>0..*</td>
<td>Information about treatment involving use of insecticides; should include the name of pesticide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple pesticide regimens</td>
<td>{text};{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0001056/">0001056</a></td>
<td>ph_regm</td>
<td>0..*</td>
<td>Information about treatment involving exposure of plants to varying levels of ph of the growth media, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimen</td>
<td>{float};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000582/">0000582</a></td>
<td>salt_regm</td>
<td>0..*</td>
<td>Information about treatment involving use of salts as supplement to liquid and soil growth media; should include the name of salt, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple salt regimens</td>
<td>{text};{float} {unit};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0001068/">0001068</a></td>
<td>season_environment</td>
<td>0..*</td>
<td>Treatment involving an exposure to a particular season (e.g. Winter, summer, rabi, rainy etc.), treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000754/">0000754</a></td>
<td>perturbation</td>
<td>0..*</td>
<td>Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</td>
<td>{text};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000116/">0000116</a></td>
<td>samp_store_dur</td>
<td>1..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
</tr>
<tr>
<td><a href="../0000755/">0000755</a></td>
<td>samp_store_loc</td>
<td>1..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000002/">0000002</a></td>
<td>samp_collect_device</td>
<td>1..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000255/">0000255</a></td>
<td>host_age</td>
<td>1..1</td>
<td>Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000248/">0000248</a></td>
<td>host_common_name</td>
<td>1..1</td>
<td>Common name of the host.</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000365/">0000365</a></td>
<td>host_genotype</td>
<td>1..1</td>
<td>Observed genotype</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000264/">0000264</a></td>
<td>host_height</td>
<td>1..1</td>
<td>The height of subject</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001318/">0001318</a></td>
<td>host_subspecf_genlin</td>
<td>0..*</td>
<td>Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
</tr>
<tr>
<td><a href="../0000256/">0000256</a></td>
<td>host_length</td>
<td>1..1</td>
<td>The length of subject</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000251/">0000251</a></td>
<td>host_life_stage</td>
<td>1..1</td>
<td>Description of life stage of host</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000874/">0000874</a></td>
<td>host_phenotype</td>
<td>1..1</td>
<td>Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000250/">0000250</a></td>
<td>host_taxid</td>
<td>1..1</td>
<td>NCBI taxon id of the host, e.g. 9606</td>
<td>{NCBI taxid}</td>
</tr>
<tr>
<td><a href="../0000263/">0000263</a></td>
<td>host_tot_mass</td>
<td>1..1</td>
<td>Total mass of the host at collection, the unit depends on host</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001058/">0001058</a></td>
<td>plant_product</td>
<td>0..1</td>
<td>Substance produced by the plant, where the sample was obtained from</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000753/">0000753</a></td>
<td>oxy_stat_samp</td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000058/">0000058</a></td>
<td>assembly_quality</td>
<td>0..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
</tr>
<tr>
<td><a href="../0000091/">0000091</a></td>
<td>url</td>
<td>0..*</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{URL}</td>
</tr>
<tr>
<td><a href="../0001083/">0001083</a></td>
<td>fao_class</td>
<td>0..1</td>
<td>Soil classification from the FAO World Reference Database for Soil Resources. The list can be found at http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000330/">0000330</a></td>
<td>local_class</td>
<td>0..1</td>
<td>Soil classification based on local soil classification system</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000331/">0000331</a></td>
<td>local_class_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the local soil classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000332/">0000332</a></td>
<td>soil_type</td>
<td>1..1</td>
<td>Description of the soil type or classification. This field accepts terms under soil (http://purl.obolibrary.org/obo/ENVO_00001998).  Multiple terms can be separated by pipes.</td>
<td>{termLabel} {[termID]}</td>
</tr>
<tr>
<td><a href="../0000334/">0000334</a></td>
<td>soil_type_meth</td>
<td>1..1</td>
<td>Reference or method used in determining soil series name or other lower-level classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0001082/">0001082</a></td>
<td>horizon</td>
<td>0..1</td>
<td>Specific layer in the land area which measures parallel to the soil surface and possesses physical characteristics which differ from the layers above and beneath</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000321/">0000321</a></td>
<td>horizon_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the horizon</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000329/">0000329</a></td>
<td>link_class_info</td>
<td>0..1</td>
<td>Link to digitized soil maps or other soil classification information</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000315/">0000315</a></td>
<td>previous_land_use</td>
<td>0..1</td>
<td>Previous land use and dates</td>
<td>{text};{timestamp}</td>
</tr>
<tr>
<td><a href="../0000316/">0000316</a></td>
<td>previous_land_use_meth</td>
<td>0..1</td>
<td>Reference or method used in determining previous land use and dates</td>
<td>{PMID</td>
</tr>
<tr>
<td><a href="../0000318/">0000318</a></td>
<td>crop_rotation</td>
<td>0..1</td>
<td>Whether or not crop is rotated, and if yes, rotation schedule</td>
<td>{boolean};{Rn/start_time/end_time/duration}</td>
</tr>
<tr>
<td><a href="../0000639/">0000639</a></td>
<td>agrochem_addition</td>
<td>0..*</td>
<td>Addition of fertilizers, pesticides, etc. - amount and time of applications</td>
<td>{text};{float} {unit};{timestamp}</td>
</tr>
<tr>
<td><a href="../0001081/">0001081</a></td>
<td>tillage</td>
<td>0..*</td>
<td>Note method(s) used for tilling</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001086/">0001086</a></td>
<td>fire</td>
<td>0..1</td>
<td>Historical and/or physical evidence of fire</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000319/">0000319</a></td>
<td>flooding</td>
<td>0..1</td>
<td>Historical and/or physical evidence of flooding</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000320/">0000320</a></td>
<td>extreme_event</td>
<td>0..1</td>
<td>Unusual physical events that may have affected microbial populations</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000328/">0000328</a></td>
<td>link_climate_info</td>
<td>0..1</td>
<td>Link to climate resource</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000642/">0000642</a></td>
<td>annual_temp</td>
<td>0..1</td>
<td>Mean annual temperature</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000643/">0000643</a></td>
<td>season_temp</td>
<td>0..1</td>
<td>Mean seasonal temperature</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000644/">0000644</a></td>
<td>annual_precpt</td>
<td>0..1</td>
<td>The average of all annual precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000645/">0000645</a></td>
<td>season_precpt</td>
<td>0..1</td>
<td>The average of all seasonal precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001080/">0001080</a></td>
<td>cur_land_use</td>
<td>0..1</td>
<td>Present state of sample site</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000646/">0000646</a></td>
<td>slope_gradient</td>
<td>0..1</td>
<td>Commonly called 'slope'. The angle between ground surface and a horizontal line (in percent). This is the direction that overland water would flow. This measure is usually taken with a hand level meter or clinometer</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000647/">0000647</a></td>
<td>slope_aspect</td>
<td>0..1</td>
<td>The direction a slope faces. While looking down a slope use a compass to record the direction you are facing (direction or degrees); e.g., nw or 315 degrees. This measure provides an indication of sun and wind exposure that will influence soil temperature and evapotranspiration.</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001084/">0001084</a></td>
<td>profile_position</td>
<td>0..1</td>
<td>Cross-sectional position in the hillslope where sample was collected.sample area position in relation to surrounding areas</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0001085/">0001085</a></td>
<td>drainage_class</td>
<td>0..1</td>
<td>Drainage classification from a standard system such as the USDA system</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000327/">0000327</a></td>
<td>store_cond</td>
<td>1..1</td>
<td>Explain how and for how long the soil sample was stored before DNA extraction (fresh/frozen/other).</td>
<td>{text};{duration}</td>
</tr>
<tr>
<td><a href="../0001106/">0001106</a></td>
<td>ph_meth</td>
<td>0..1</td>
<td>Reference or method used in determining ph</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000312/">0000312</a></td>
<td>cur_vegetation</td>
<td>0..1</td>
<td>Vegetation classification from one or more standard classification systems, or agricultural crop</td>
<td>{text}</td>
</tr>
<tr>
<td><a href="../0000314/">0000314</a></td>
<td>cur_vegetation_meth</td>
<td>0..1</td>
<td>Reference or method used in vegetation classification</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000533/">0000533</a></td>
<td>tot_org_carb</td>
<td>0..1</td>
<td>Definition for soil: total organic carbon content of the soil, definition otherwise: total organic carbon content</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000337/">0000337</a></td>
<td>tot_org_c_meth</td>
<td>0..1</td>
<td>Reference or method used in determining total organic carbon</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000102/">0000102</a></td>
<td>tot_nitro</td>
<td>0..1</td>
<td>Total nitrogen concentration of water samples, calculated by: total nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be measured without filtering, reported as nitrogen</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000338/">0000338</a></td>
<td>tot_n_meth</td>
<td>0..1</td>
<td>Reference or method used in determining the total nitrogen</td>
<td>{PMID</td>
</tr>
<tr>
<td><a href="../0000650/">0000650</a></td>
<td>microbial_biomass</td>
<td>1..1</td>
<td>The part of the organic matter in the soil that constitutes living microorganisms smaller than 5-10 micrometer. If you keep this, you would need to have correction factors used for conversion to the final units</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000339/">0000339</a></td>
<td>microbial_biomass_meth</td>
<td>1..1</td>
<td>Reference or method used in determining microbial biomass</td>
<td>{PMID</td>
</tr>
<tr>
<td><a href="../0000343/">0000343</a></td>
<td>heavy_metals_meth</td>
<td>0..1</td>
<td>Reference or method used in determining heavy metals</td>
<td>{PMID}</td>
</tr>
<tr>
<td><a href="../0000525/">0000525</a></td>
<td>tot_car</td>
<td>0..1</td>
<td>Total carbon content</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000689/">0000689</a></td>
<td>tot_phos</td>
<td>0..1</td>
<td>Total amount or concentration of phosphate</td>
<td>None</td>
</tr>
<tr>
<td><a href="../0000322/">0000322</a></td>
<td>sieving</td>
<td>1..1</td>
<td>Collection design of pooled samples and/or sieve size and amount of sample sieved</td>
<td>{{text}</td>
</tr>
<tr>
<td><a href="../0000325/">0000325</a></td>
<td>pool_dna_extracts</td>
<td>1..1</td>
<td>Indicate whether multiple DNA extractions were mixed. If the answer yes, the number of extracts that were pooled should be given</td>
<td>{boolean};{integer}</td>
</tr>
<tr>
<td><a href="../0000752/">0000752</a></td>
<td>misc_param</td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
</tr>
</tbody>
</table>

              
            </article>
            
          </div>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    <script id="__config" type="application/json">{"base": "..", "features": [], "search": "../assets/javascripts/workers/search.b97dbffb.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.0238f547.min.js"></script>
      
    
  </body>
</html>